Sources/MongoSwift/APM.swift

```diff
@@ -132,7 +132,7 @@ public struct CommandFailedEvent: MongoEvent, InitializableFromOpaquePointer {
         self.commandName = String(cString: mongoc_apm_command_failed_get_command_name(event))
         var error = bson_error_t()
         mongoc_apm_command_failed_get_error(event, &error)
-        self.failure = MongoError.commandError(message: toErrorString(error))
+        self.failure = parseMongocError(error: error) // should always return a ServerError.commandError
```

Comment:
nit: should `parseMongocError` s first parameter not be named? Seems redundant to have `error: error` everywhere, since the method also tells us its parsing an error